Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:3.8-slim-buster as builder
+
+ENV POETRY_VERSION=1.0.10
+
+# install poetry
+RUN pip install "poetry==$POETRY_VERSION"
+
+# set path of workdir in docker
+WORKDIR /app
+
+# copy my current dir to /app on WORKDIR
+COPY . /app
+
+# step to show my list of copied files and subdirectories
+RUN ls -la ./
+
+# turn off virtual envir creation from poetry as we're in docker
+# install poetry dependencies #COPY pyproject.toml poetry.lock ./
+RUN poetry config virtualenvs.create false \
+    && poetry install --no-interaction
```

Comment:
You have the commented out line "COPY pyproject.toml poetry.lock ./"

If include that, then `COPY  . /app` can be moved after poetry install, as it's more likely to change. So you'll save time when rebuilding your image after code changes.